fix: Added utils to installed packages

SRFU-NN opened the pull request:
pip install of version 0.8.2 can not be imported since the package ProcessOptimizer.utils is not included.

When @SRFU-NN changed utils.py to a folder, he forgot to add it to setup.py.

SRFU-NN committed:
fix: Added utils to installed packages

SRFU-NN committed:
doc: Updated changelog

SRFU-NN committed:
CI: Testing in python 3.11

SRFU-NN committed:
CI: testing bleeding edge in python 3.11

sqbl commented:
We use scipy 1.9.1 in our stable-install. And guess what… :-)
Scipy added wheels for python 3.11 in version 1.9.2.

Suggested solution: upgrade the versions for the stable version. But lets discuss with users of BrownieBeeAPI

SRFU-NN commented:
Good suggestion. Let’s remove Python 3.11 from the BB testing, so we can get the new release done now.

Another solution is to only test BB on one python version - if everything else is locked down, why not the Python version?

Which version of script should we upgrade to? The latest, the smallest upgrade, something in between?

Sendt fra min iPhone

Den 17. aug. 2023 kl. 21.29 skrev Søren Bertelsen ***@***.***>:

﻿

Caution: This email originated from outside of the Novo Nordisk email system. Use caution when clicking on links or opening attachments.


We use scipy 1.9.1 in our stable-install. And guess what… :-)
Scipy added wheels for python 3.11 in version 1.9.2.

Suggested solution: upgrade the versions for the stable version. But lets discuss with users of BrownieBeeAPI

—
Reply to this email directly, view it on GitHub<https://github.com/novonordisk-research/ProcessOptimizer/pull/199#issuecomment-1682842580>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AY4V7XI6VEC6MI3XQFMBGZLXVZWGRANCNFSM6AAAAAA3T5IUDM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SRFU-NN committed:
CI, fix: Stopped testing BB with Python 3.11

Brownie Bee uses scipy v. 1.9.1, which doesnøt support Python 3.11.
A long term solution is either to update the package verion in BB,
or to only test BB with one Python version.

dk-teknologisk-mon reviewed (approved):
Looks fine to me. As for which python version BB uses, I think it is best for now to just leave it checking up to python 3.10, and then I will figure out if/when we can add this version there too. As younsay, the UI must presuppose some version with how it is configured.
